projects/hy-p-manual.diff

```diff
@@ -331,19 +349,61 @@ index 61a8231..c30a628 100644
  
  // Builder konfigurieren
  // - Zuweisung der Geometrie auf Ebene von geometry zulassen
-@@ -4132,12 +4045,41 @@ Die lokale ID wird aus dem Namen der Ziel-Objektart, der AAA-ObjektID des Quell-
-             </core:message>
+@@ -4110,11 +4023,40 @@ Die lokale ID wird aus dem Namen der Ziel-Objektart, der AAA-ObjektID des Quell-
          </annotation>
      </cell>
--    <cell relation="custom:alignment:adv.inspire.AX_Lagebezeichnung.GeographicalName" id="Cd6f3ce91-ebb8-4699-aa50-0056bba3c68e" priority="normal">
+     <cell relation="custom:alignment:adv.inspire.AX_Lagebezeichnung.GeographicalName" id="C45a9e47b-e2c7-408a-a3b8-3e43593337e0" priority="normal">
 -        <source name="lagebezeichnung">
-+    <cell relation="custom:alignment:adv.inspire.AX_Lagebezeichnung.GeographicalName" id="Cd3870b33-cff0-4f43-a422-d73ab401da15" priority="normal">
 +        <source name="lage_gemeinde">
++            <property>
++                <type name="ax_stehendesgewaesser" ns="jdbc:postgresql:testbed:aaa_ogr"/>
++                <child name="gemeinde"/>
++            </property>
++        </source>
++        <source name="lage_kreis">
++            <property>
++                <type name="ax_stehendesgewaesser" ns="jdbc:postgresql:testbed:aaa_ogr"/>
++                <child name="kreis"/>
++            </property>
++        </source>
++        <source name="lage_lage">
              <property>
--                <type name="AX_StehendesGewaesserType" ns="http://www.adv-online.de/namespaces/adv/gid/6.0"/>
+-                <type name="AX_HafenbeckenType" ns="http://www.adv-online.de/namespaces/adv/gid/6.0"/>
 -                <child name="name" ns="http://www.adv-online.de/namespaces/adv/gid/6.0"/>
 -                <child name="AX_Lagebezeichnung" ns="http://www.adv-online.de/namespaces/adv/gid/6.0"/>
 +                <type name="ax_stehendesgewaesser" ns="jdbc:postgresql:testbed:aaa_ogr"/>
```

Comment:
If I understand it correctly "AX_Hafenbecken" is replaced with "ax_stehendesgewaesser" now if the diff is applied. Is that intended as it is a different type?

Reply:
Good catch of a copy-paste mistake! `AX_Hafenbecken` needs to be replaced with `ax_hafenbecken` of course.